src/hardwareio/IMUCalibDumper/src/IMUCalibDumper.cpp

```diff
@@ -0,0 +1,163 @@
+/*******************************************************************************
+                                                                               *
+ * Copyright (C) 2020 Fondazione Istituto Italiano di Tecnologia (IIT)         *
+ * All Rights Reserved.                                                        *
+                                                                               *
+                                                                              */
+
+/**
+ * @file: IMUCalibDumper/src/IMUCalibDumper.cpp
+ * @authors:    Leandro de Souza Rosa <leandro.desouzarosa@iit.it>
+ */
+
+#include "IMUCalibDumper.h"
+
+bool IMUCalibDumper::configure(yarp::os::ResourceFinder& rf)
+{
+    //set the module name used to name ports
+    setName((rf.check("name", Value("/IMUCalibDumper")).asString()).c_str());
+
+    //open io ports
+    if(!imu_port.open(getName() + "/IMU:i")) {
+        yError() << "Could not open input port";
+        return false;
+    }
+
+    //imu calibration
+    Bottle *imucalibration = rf.find("imu_calib").asList();
+    if(imu_helper.configure(imucalibration)) {
+        yInfo() << "IMU calibration:" << imucalibration->toString();
+    }
+
```

Comment:
I don't see the calibration of the IMU being used. is it a bug? should this instead be commented out?